make install fails because flake8 dependency is not installed

Create a new env and run `make install`

```
$ make clean install
rm -fr build/
rm -fr dist/
rm -fr .eggs/
find . -name '*.egg-info' -exec rm -fr {} +
find . -name '*.egg' -exec rm -f {} +
find . -name '*.pyc' -exec rm -f {} +
find . -name '*.pyo' -exec rm -f {} +
find . -name '*~' -exec rm -f {} +
find . -name '__pycache__' -exec rm -fr {} +
rm -fr .tox/
rm -f .coverage
rm -fr htmlcov/
rm -fr .pytest_cache
flake8 kfp_notebook etc
make: flake8: No such file or directory
make: *** [lint] Error 1

```

--- a/Makefile
+++ b/Makefile
@@ -56,7 +56,7 @@ clean-test:
 	rm -fr htmlcov/
 	rm -fr .pytest_cache
 
-lint: ## check style with flake8
+lint: test-dependencies ## check style with flake8
 	flake8 kfp_notebook etc
 
 test-dependencies:

--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -6,3 +6,4 @@ nbconvert
 papermill
 pytest
 pytest_virtualenv
+flake8>=3.5.0,<3.9.0

